### ISSUE TITLE
IDA plugin search filter not working

### ISSUE THREAD

**Issue by mr-tz** [fireeye/capa#299]
### Description

Filter gives no results, including searches for rule name or namespace.

![2020-09-09_12-40-29](https://user-images.githubusercontent.com/17606537/92589049-43869400-f29a-11ea-8614-d6f4df560f2c.png)

![2020-09-09_12-40-22](https://user-images.githubusercontent.com/17606537/92589059-45e8ee00-f29a-11ea-81cd-8d2515bf9595.png)

### Versions

IDA Version 7.5.200728 Windows x64

```
Python 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)] 
IDAPython v7.4.0 final (serial 0) (c) The IDAPython Team <idapython@googlegroups.com>
```

**Comment by williballenthin** [comment]
hm, i can't reproduce this :-(

**Comment by williballenthin** [comment]
for a moment i could, but now i can't. wonder if its because i pulled from master since then?

**Comment by mike-hunhoff** [comment]
I reproduced in:

```
Python 2.7.16 (v2.7.16:413a49145e, Mar  4 2019, 01:37:19) [MSC v.1500 64 bit (AMD64)] 
IDAPython v7.4.0 final (serial 0) (c) The IDAPython Team <idapython@googlegroups.com>
```

The search proxy fails because of the following line:
https://github.com/fireeye/capa/blob/146caed7aad1d1580b53eb4d128bfaea770f5cc6/capa/ida/plugin/proxy.py#L211

Because the data types are `unicode` not `str`:

```
(u'f', u'access PEB ldr_data (2 matches)')
(<type 'unicode'>, <type 'unicode'>)
```

**Comment by mike-hunhoff** [comment]
addressed in #302
